lib/file_store/adapters/s3.ex

```diff
@@ -151,6 +157,18 @@ if Code.ensure_loaded?(ExAws.S3) do
         |> Stream.map(fn file -> file.key end)
       end
 
+      defp maybe_set_content_disposition(opts) do
```

Comment:
Could we instead add the following function to `FileStore.Utils`?

```elixir
def rename_key(opts, key, new_key) do
  case Keyword.fetch(opts, key) do
    {:ok, value} ->
      opts
      |> Keyword.delete(key)
      |> Keyword.put(new_key, value)

    :error ->
      opts
  end
end
```

Then, you could say:

```elixir
opts =
  opts
  |> Keyword.take([:content_type, :disposition])
  |> Utils.rename_key(:disposition, :content_disposition)
```

It might be nice for adapters to have that function available, as it's common for this library to forward args.

Reply:
Good idea. Threw up some tests for it as well